### PR TITLE
feat/CWALL-252

### PR DESCRIPTION
**Pull request by BtencateSphereon**

### CODE REVIEW

File: packages/web-wallet/src/dataProviders/identifiersDataProvider.ts [hunk 1]
```diff
@@ -137,7 +137,7 @@ export const identifiersDataProvider = (): DataProvider => ({
         clientId,
         credentialIssuer: ebsi?.tao?.url!,
         jwksUri,
-        environment: network as EbsiEnvironment,
+        environment: network as EbsiEnvironment, //FIXME this is casting a possible undefined where environment is mandatory
```
Comment (sanderPostma) [inline packages/web-wallet/src/dataProviders/identifiersDataProvider.ts:140]: Shouldn't you throw an error then when (under method === 'ebsi') network is undefined? Or should we really be able to pass as undefined?

File: packages/web-wallet/src/components/views/CredentialsList/index.tsx [hunk 2]
```diff
@@ -343,7 +337,17 @@ const CredentialsList: FC<Props> = (props: Props): ReactElement => {
...
         },
         {
-          onSuccess: () => refetchParties().then(() => onCloseImportCredentialModal())
+          onSuccess: async () => {
+            if (correlationId.toLowerCase().startsWith('did:ebsi') && uniformCredential.type.includes('VerifiableAuthorisationToOnboard')) {
+              // We want to call the register in the background, so for now we are not dealing with the result, we just execute the register function
+              void registerDidEbsiOnLedger({
+                did: correlationId,
+                credentialIssuer: issuerName
+              }).catch(() => console.log(`Unable to register ebsi did ${correlationId} for issuer ${issuerName}`))
```
Comment (sanderPostma) [inline packages/web-wallet/src/components/views/CredentialsList/index.tsx:346]: Shouldn't we propagate this error?

**Review by sanderPostma — CHANGES_REQUESTED** [review]

File: packages/web-wallet/src/dataProviders/identifiersDataProvider.ts (hunk 1 above)
Comment (BtencateSphereon) [reply packages/web-wallet/src/dataProviders/identifiersDataProvider.ts:140]: no idea what we want here, i just noticed this and placed a fixme

File: packages/web-wallet/src/components/views/CredentialsList/index.tsx (hunk 2 above)
Comment (BtencateSphereon) [reply packages/web-wallet/src/components/views/CredentialsList/index.tsx:346]: it was stated we want this process to be in the background. i have no ui for unhappy flow. so fow now it just logs. i have no idea what the unhappy path should do.

i can imagine we want to register on import, if it fails we have options later in the ui to register again if needed

**Review by sanderPostma — APPROVED** [review]
LGTM

**Pull request merged**